controller/src/main/java/io/pravega/controller/store/stream/tables/HistoryRecord.java

```diff
@@ -38,6 +38,16 @@
 
     @Getter
     private final int epoch;
+    /**
+     * This refers to epoch whose duplicate this epoch HistoryRecord may be.
```

Comment:
`... refers to the epoch... `

Reply:
Perhaps a better rephrasing of this sentence would be: `The reference epoch is the original epoch that this current epoch duplicates.`